projects/aca-shared/src/lib/components/open-in-app/open-in-app.component.scss

```diff
@@ -0,0 +1,15 @@
+.container{
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+.mat-dialog-container{
+  padding: 12px;
+  border-radius: 36px;
+  background-color: var(--theme-blue-button-color);
+  color: var(--theme-about-panel-background-color);
+}
+.material-icons{
+  font-size: 16px;
```

Comment:
why do you need this change?

Reply:
removed the font of icon back to original size.

Reply:
removed the font-size and made it to original size